appletree/component.py

```diff
@@ -468,15 +502,10 @@ def simulate_weighed_data(self,
 @export
 def add_component_extensions(module1, module2):
     """Add components of module2 to module1"""
-    for x in dir(module2):
-        x = getattr(module2, x)
-        if not isinstance(x, type(type)):
-            continue
-        _add_component_extension(module1, x)
+    utils.add_extensions(module1, module2, Component)
 
 
 @export
```

Comment:
Maybe I missed something before. Why do we wanna export a private function?

Reply:
This is a mistake completely.